backend/tests/integration-async/conftest.py

```diff
@@ -31,6 +31,7 @@ def app():
     app_config["CELERY_TASK_ALWAYS_EAGER"] = False
     app_config["CELERY_TASK_EAGER_PROPAGATES"] = False
     app_config["TESTING"] = True
+    app_config["FEATURE_FLAG_ENABLE_AUTHOR_DISAMBIGUATION"] = True
```

Comment:
I think it's better to do it in the testcases, otherwise it's hard to follow. Please also test the feature flag itself (nothing happens when feature is disabled).

Reply:
But then how we can make sure that it is no breaking for some reason other tests?

Reply:
I don't think I understand the question. If the feature flag is false in the `config.py`, you should explicitly opt-in to enable the feature flags in the tests where you need to test behavior of the feature. Then it's clear what depends on the feature flag (you can look at the other tests where we need to change some feature flags).